core/support.go

```diff
@@ -55,29 +142,38 @@ func NewSupportClient() *SupportClientImpl {
 	}
 }
 
+// RequestServiceLimitsRefreshLoop ...
 func (client *SupportClientImpl) RequestServiceLimitsRefreshLoop() {
-	input := &support.RefreshTrustedAdvisorCheckInput{
-		CheckId: aws.String(checkId),
-	}
+	var (
+		waitMs        int64
+		refreshStatus string
+	)
 
 	for {
-		glog.Infof("Refreshing Trusted Advisor check '%s'...", checkId)
-		output, err := client.SupportClient.RefreshTrustedAdvisorCheck(input)
-		if err != nil {
-			glog.Errorf("Error when requesting status refresh: %v", err)
-			continue
+		for _, checkID := range checkIDs {
+			input := &support.RefreshTrustedAdvisorCheckInput{
+				CheckId: aws.String(checkID),
+			}
+			glog.Infof("Refreshing Trusted Advisor check '%s'...", checkID)
+			output, err := client.SupportClient.RefreshTrustedAdvisorCheck(input)
+			if err != nil {
+				glog.Errorf("Error when requesting status refresh: %v", err)
+				continue
+			}
+
+			waitMs = *output.Status.MillisUntilNextRefreshable
+			refreshStatus = aws.StringValue(output.Status.Status)
 		}
-
-		waitMs := *output.Status.MillisUntilNextRefreshable
-
-		glog.Infof("Refresh status is '%s', waiting %dms until the next refresh...", aws.StringValue(output.Status.Status), waitMs)
+		glog.Infof("Refresh status is '%s', waiting %d minutes until the next refresh...", refreshStatus, waitMs/60000)
```

Comment:
We are reporting only the status of the last check, which seems misleading.

Since we are already reporting the error for a specific check inside the second `for`, maybe this log line is not providing any useful information and thus can be removed?

```suggestion
```

Reply:
I think its still useful to have the refresh status output, I'm not sure about the duration, I think it does not hurt to leave it there so anyone looking at the program logs can easily see the program is just waiting for its next iteration.

Reply:
Makes sense, although I'd remove the refresh status from the message, as it's just the refresh status for the last check and might not reflect the status of all checks.

Reply:
agree, I added a commit to remove the refresh status., and only logs we are waiting 60 min for the next iteration:
`I1121 09:03:23.834109     536 support.go:156] Refreshing Trusted Advisor check 'kM7QQ0l7J9'...
I1121 09:03:23.990282     536 support.go:156] Refreshing Trusted Advisor check 'jL7PP0l7J9'...
I1121 09:03:24.099742     536 support.go:163] Waiting 60 minutes until the next refresh...`